README.md

```diff
@@ -11,6 +11,20 @@
 
 See [catalogue-acceptance-tests]("https://github.com/hmrc/catalogue-acceptance-tests") for help setting up dependent services.
 
+Run docker with the following command 
```

Comment:
This is already documented in the above `catalogue-acceptance-tests`, maybe could just reword "for help setting up dependent services and mongo"

Reply:
I think this might be backwards: we should have some details about how to run/test the catalogue-frontend locally in this README rather than in the acceptance tests.  Maybe a wider discussion for later about making development easier :)

Reply:
Removed duplicate docs